modules/backend/formwidgets/mediafinder/assets/js/mediafinder.js

```diff
@@ -41,6 +41,10 @@
             this.options.isImage = this.$el.hasClass('is-image')
         }
 
+        if (this.options.mode === null) {
+            this.options.mode = this.$el.data("mediafinder-mode") || "all";
```

Comment:
Our JS style guide says single quotes for strings.
```suggestion
            this.options.mode = this.$el.data('mediafinder-mode') || 'all';
```